src/store/metadata/stub/content/editions/adult-obesity-prevalence-aged-18-years-and-over_2023-05.json

```diff
@@ -0,0 +1,155 @@
+                "titles": "Area Name",
+                "description": "The name assiciated with The geography code "
+              },
+              {
+                "name": "geography",
+                "datatype": "string",
+                "titles": "Geography Type",
+                "description": "The categorisation of the geography code"
+              },
+              {
+                "name": "variable_name",
+                "datatype": "string",
+                "titles": "Variable Name",
+                "description": "Which ESS variable is being measured"
+              },
+              {
+                "name": "indicator", 		
+                "datatype": "string",
+                "titles": "Indicator",
+                "description": "Which ESS indicator is being measured"
+              },
+              {
+                "name": "mission",
+                "datatype": "string",
+                "titles": "Mission",
+                "description": "Which ESS mission number this dataset is fulfilling"
+              },
+              {
+                "name": "category",
+                "datatype": "string",
+                "titles": "Category",
+                "description": "Which ESS category the dataset belongs to"
+              },
+              {
+                  "name": "observation",
+                  "datatype": "decimal",
+                  "titles": "Observation",
+                  "description": "Observation value for the chosen variable"
+                },
+                {
+                    "name": "measure",
+                    "datatype": "string",
+                    "titles": "Measure Type",
+                    "description": "Measure Type of the Observational Value"
+                },
+                {
+                    "name": "unit",
+                    "datatype": "string",
+                    "titles": "Unit",
+                    "description": "Unit Type of the Observational Value"
+                  },
+                {
+                "name": "lower_confidence_Interval_95%",
+                "datatype": "decimal",
+                "titles": "Lower Confidence Interval (95%)",
+                "description": "The interval estimate gives an indication of how much uncertainty there is in our estimate of the true mean, Lower bound"
+              },
+              {
+                "name": "upper_confidence_Interval_95%",
+                "datatype": "decimal",
+                "titles": "Upper Confidence Interval (95%)",
+                "description": "The interval estimate gives an indication of how much uncertainty there is in our estimate of the true mean, Upper bound"
+              },
+              {
+                "name": "observation_status",
+                "datatype": "string",
+                "titles": "Observation Status",
+                "description": "Additional Context to the Observation if needed"
+              },
+              {
```

Comment:
Please delete `polarity` and `notes` column mappings.
They are set to false on the json file on family-ess repo and they are not displayed on the PMD-Beta site, for which both links are indicated earlier